runtime/doc/builtin.txt

```diff
@@ -7274,7 +7274,9 @@ matchfuzzy({list}, {str} [, {dict}])			*matchfuzzy()*
 				that contain the characters in {str} in the
 				given sequence.
 		    limit	Maximum number of matches in {list} to be
-				returned.  Zero means no limit.
+				returned. Zero means no limit.
+		    camelcase	Apply score bounds when matching CamelCase
+				pattern. Default is v:true.
```

Comment:
What about:

Use enhanced camel case scoring making results better suited completion related to programming languages.